internal/pkg/cli/app_init.go

```diff
@@ -80,6 +81,7 @@ func newInitAppOpts(vars initAppVars) (*initAppOpts, error) {
 		prompt:           prompt.New(),
 		prog:             termprogress.NewSpinner(log.DiagnosticWriter),
 		iam:              iam.New(sess),
+		iamRoleManager:   iam.New(sess),
```

Comment:
Can we reuse the existing session instead of creating new one? for example -
```
                 iam := iam.New(sess),
                 iam: iam,
		 iamRoleManager:  iam,
```